src/screens/Game/GamePage/index.css

```diff
@@ -69,40 +64,55 @@
     -webkit-box-orient: vertical;
 }
 
-.installProgress[value] {
-    appearance: none;
-    width: 169px;
-    height: 5px;
+.gameInfo,
+.gameTools,
+.gameRequirements {
+    display: none;
+    align-self: flex-start;
+    padding: 8vh 2.2em 1.2em 1.2em;
 }
 
-.installProgress[value]::-webkit-progress-value {
-    color: var(--success);
-    animation: animate-stripes 5s linear infinite;
+.gameTabs {
+    flex: 60%;
+    flex-shrink: 0;
+    flex-grow: 0;
+    position: relative;
+    display: grid;
+    grid-template-rows: min-content 1fr;
 }
 
-.requirements {
-    margin-top: 2em;
+.gameTabs.infoTab .gameInfo,
+.gameTabs.toolsTab .gameTools,
+.gameTabs.requirementsTab .gameRequirements {
+    display: flex;
 }
 
-.requirements table td {
-    width: 33.33%;
-    vertical-align: baseline;
-    padding: 0.5em;
+.gameTabs nav {
+    text-align: right;
+    padding: 15px;
 }
-
-table td.specs {
+.gameTabs nav button {
+    margin-right: 1em;
+    border: none;
+    background: none;
+    color: var(--text-primary);
+    cursor: pointer;
+}
+.gameTabs.infoTab button[data-tab='info'],
+.gameTabs.toolsTab button[data-tab='tools'],
+.gameTabs.requirementsTab button[data-tab='requirements'] {
     color: var(--secondary);
-    opacity: 0.8;
 }
 
-.requirements table span.title {
-    color: var(--text-secondary);
-    font-size: 1em;
+.installProgress[value] {
```

Comment:
from this line down, github is acting funny with the diff, I didn't change this lines, though I moved the requirements css to another file